src/traces/table/plot.js

```diff
@@ -0,0 +1,892 @@
+                }
+                if(currentRowLength) {
+                    d.value += currentRow.join(c.wrapSpacer);
+                }
+                d.wrapped = true;
+            });
+
+        // the pre-wrapped text was rendered only for the text measurements
+        cellTextHolder.selectAll('tspan.line').remove();
+
+        // resupply text, now wrapped
+        populateCellText(cellTextHolder.select('.cellText'), tableControlView, columnBlock);
+        d3.select(element.parentNode.parentNode).call(setCellHeightAndPositionY);
+    };
+}
+
+function updateYPositionMaker(columnBlock, element, tableControlView, gd, d) {
+    return function updateYPosition() {
+        if(d.settledY) return;
+        var cellTextHolder = d3.select(element.parentNode);
+        var l = getBlock(d);
+        var rowIndex = d.key - l.firstRowIndex;
+
+        var declaredRowHeight = l.rows[rowIndex].rowHeight;
+
+        var requiredHeight = d.cellHeightMayIncrease ? element.parentNode.getBoundingClientRect().height + 2 * c.cellPad : declaredRowHeight;
+
+        var finalHeight = Math.max(requiredHeight, declaredRowHeight);
+        var increase = finalHeight - l.rows[rowIndex].rowHeight;
+
+        if(increase) {
+
+            // current row height increased
+            l.rows[rowIndex].rowHeight = finalHeight;
+
+            columnBlock
+                .selectAll('.columnCell')
+                .call(setCellHeightAndPositionY);
+
+            updateBlockYPosition(null, columnBlock.filter(cellsBlock), 0);
+
+            // if d.column.type === 'header', then the scrollbar has to be pushed downward to the scrollable area
+            // if d.column.type === 'cells', it can still be relevant if total scrolling content height is less than the
+            //                               scrollable window, as increases to row heights may need scrollbar updates
+            renderScrollbarKit(tableControlView, gd);
+        }
+
+        cellTextHolder
+            .attr('transform', function() {
+                // this code block is only invoked for items where d.cellHeightMayIncrease is truthy
+                var element = this;
+                var columnCellElement = element.parentNode;
+                var box = columnCellElement.getBoundingClientRect();
+                var rectBox = d3.select(element.parentNode).select('.cellRect').node().getBoundingClientRect();
+                var currentTransform = element.transform.baseVal.consolidate();
+                var yPosition = rectBox.top - box.top + (currentTransform ? currentTransform.matrix.f : c.cellPad);
+                return 'translate(' + xPosition(d, d3.select(element.parentNode).select('.cellTextHolder').node().getBoundingClientRect().width) + ' ' + yPosition + ')';
+            });
+
+        // MathJax styling has to be killed for HTML DOM color specifications to seep through
+        cellTextHolder.selectAll('svg').selectAll('*')
```

Comment:
Looks hacky. Can we do better?

Reply:
in regular `convertToTspans` MathJax gets colored [here](https://github.com/plotly/plotly.js/blob/master/src/lib/svg_text_utils.js#L118) based on the color that was already applied to the text it was converted from - does that not apply here?

Reply:
I'll check your link, maybe it solves the need for the current hack, I wasn't happy it drew in black when `fill` was white but thought maybe there's a better way.

Reply:
probably just an order-of-operations issue, we assume the text is fully styled before being converted to MathJax. It would probably be good for us to break this out like we did with `positionText` so it can be applied either before or after conversion.

Reply:
Ah, thanks for the lead!

Reply:
@alexcjohnson @etpinard Fully agree w/ the order of operations (LaTeX coloring), a quick initial look didn't lead to a 5-minute solution, plan to do it tomorrow. Other tasks are, 

- ensuring `table` works fine with `Plotly.restyle`, and 
- more `default`ing shouls happens (eg. now the user always needs to specify column order) 

which seem more pressing than internal changes.

Reply:
Ok. Please add item to https://github.com/plotly/plotly.js/issues/2056